static/js/composebox_typeahead.js

```diff
@@ -365,6 +365,10 @@ exports.slash_commands = [
         text: i18n.t("/day (Toggle day mode)"),
         name: "day",
     },
+    {
+        text: i18n.t("/digress (Shift conversation to another topic)"),
```

Comment:
We should inform user about the format in which we expect the new stream and topic is to be supplied. We should discuss this in czo.

Reply:
Updated this typeahed to be `/digress #stream name>topic name (Shift conversation to another topic)`